samples/excel/45-named-item/create-and-use-named-item-for-range.yaml

```diff
@@ -0,0 +1,180 @@
+id: excel-create-and-use-named-range-item
+name: Create and use named range item
+description: Create and use a named range item
+author: OfficeDev
+host: EXCEL
+api_set:
+    ExcelApi: 1.4
+script:
+    content: |-
+        $("#add-name").click(() => tryCatch(addName));
+        $("#format-named-range").click(() => tryCatch(formatRangeByName));
+        $("#setup").click(() => tryCatch(setup));
+
+        async function addName() {
+            await Excel.run(async (context) => {
+
+                const sheet = context.workbook.worksheets.getItem("Sample");
+                const headerRange = sheet.getRange("A1:E1");
+
+                sheet.names.add("ExpensesHeader", headerRange);
```

Comment:
I'll make these fixes, but FYI: **all** of your comments (including the blank lines) are a result of me just copying code from other samples. I presume that I won't be the only contributor who does that. People will take existing, accepted code as authoritative.